shared/desktop/app/main-window.js

```diff
@@ -62,7 +63,10 @@ export default function() {
 
   const openedAtLogin = app.getLoginItemSettings().wasOpenedAtLogin
   const isRestore = getenv.boolish('KEYBASE_RESTORE_UI', false) || app.getLoginItemSettings().restoreState
-  const hideWindowOnStart = getenv.string('KEYBASE_START_UI', '') === 'hideWindow'
+  // Start not shown by default on Windows
+  const hideWindowOnStart =
```

Comment:
isn't this logic above supposed to be generic? isn't this really supposed to keep your last state and if there is none its supposed to default to hidden